services/src/main/scala/cromwell/services/metadata/WorkflowQueryKey.scala

```diff
@@ -116,7 +134,7 @@ sealed trait WorkflowQueryKey[T] {
 sealed trait DateTimeWorkflowQueryKey extends WorkflowQueryKey[Option[OffsetDateTime]] {
   override def validate(grouped: Map[String, Seq[(String, String)]]): ErrorOr[Option[OffsetDateTime]] = {
     valuesFromMap(grouped) match {
-      case vs if vs.size > 1 =>
+      case vs if vs.lengthCompare(1) > 0 =>
```

Comment:
why?

Reply:
The primary community argument is that in general `size` shouldn't be used to compare sequence lengths. That's what `lengthCompare` is specifically designed for:
- https://pavelfatin.com/scala-collections-tips-and-tricks/#sequences-length

However, it is also acknowledged that `lengthCompare` while being faster is harder to read:
- github.com/scala/collection-strawman/issues/338